### PR TITLE
Consumer Limits Stream Configuration

### PR DESCRIPTION
**Pull request by scottf**

### CODE REVIEW

File: src/main/java/io/nats/client/api/ConsumerLimits.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+         * @return Builder
+         */
+        public Builder maxAckPending(long maxAckPending) {
+            this.maxAckPending = normalize(maxAckPending, STANDARD_MIN);
+            return this;
+        }
+
+        /**
+         * Build a Placement object
+         * @return the Placement
```
Comment (piotrpio) [inline src/main/java/io/nats/client/api/ConsumerLimits.java:131]:
```suggestion
         * Build a ConsumerLimits object
         * @return ConsumerLimits
```

Comment (scottf) [reply src/main/java/io/nats/client/api/ConsumerLimits.java:131]: yeah will fix, a lot of cut paste going on

**Review by aricart — APPROVED** [review]
LGTM

**Review by piotrpio — APPROVED** [review]
LGTM!

**Pull request merged**